cmd/validate/image.go

```diff
@@ -177,10 +178,11 @@ func validateImageCmd(validate imageValidationFunc) *cobra.Command {
 		PreRunE: func(cmd *cobra.Command, args []string) (allErrors error) {
 			ctx := cmd.Context()
 			if s, err := applicationsnapshot.DetermineInputSpec(ctx, applicationsnapshot.Input{
-				File:     data.filePath,
-				JSON:     data.input,
-				Image:    data.imageRef,
-				Snapshot: data.snapshot,
+				File:           data.filePath,
+				JSON:           data.input,
+				Image:          data.imageRef,
+				Snapshot:       data.snapshot,
+				SnapshotSource: data.snapshotSource,
```

Comment:
There's a conditional required flag check at the end of this file that I think needs to be updated to take this into account, 

```
if len(data.input) > 0 || len(data.filePath) > 0 {
		if err := cmd.MarkFlagRequired("image"); err != nil {
			panic(err)
		}
	}
```